.github/workflows/master_build_merge.yml

```diff
@@ -0,0 +1,184 @@
+---
+name: Tessell Pipelines Code Build Post merge
+on:
+  workflow_call:
+    inputs:
+      type:
+        description: 'This is used to determine build type'
+        required: true
+        type: string
+      tag:
+        description: 'Latest tag for upload'
+        required: true
+        type: string
+      label:
+        description: 'Release label for upload'
+        required: true
+        type: string
+
+jobs:
+
+  build-amd:
+    if: ${{inputs.type != 'arm' }}
+    runs-on: self-hosted
+    env:
+      LATEST_TAG: ${{inputs.tag}}
+      LABEL: ${{inputs.label}}
+      DOCKERHUB_ORG: tessellprod
+      REPO: "${{github.event.pull_request.base.repo.name}}"
```

Comment:
Repo name will be available by default under env variable `GITHUB_REPOSITORY`;

Reply:
That contains TessellDevelopment/REPo so to avoid the hassle to split then use